inc/nginx-helper.php

```diff
@@ -1,8 +1,11 @@
 <?php
+
+declare(strict_types=1);
```

Comment:
The `Doctrine` coding standard is probably a bit to strict for this project.

With a lack of tests, I would probably not apply this standard.